Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/jvm/main.kt

```diff
@@ -174,28 +158,27 @@ private fun findFilesByGlobs(roots: List<Path>, globs: List<String>): Map<Path,
 
 
 private fun processCLib(args: Array<String>): Array<String>? {
-
-    val arguments = parseCommandLine(args, CInteropArguments())
-    val userDir = System.getProperty("user.dir")
-    val ktGenRoot = arguments.generated ?: userDir
-    val nativeLibsDir = arguments.natives ?: userDir
-    val flavorName = arguments.flavor ?: "jvm"
+    val argParser = ArgParser(getCInteropArguments())
+    if (!argParser.parse(args))
+        return null
+    val ktGenRoot = argParser.get<String>("generated")
+    val nativeLibsDir = argParser.get<String>("natives")
+    val flavorName = argParser.get<String>("flavor")
     val flavor = KotlinPlatform.values().single { it.name.equals(flavorName, ignoreCase = true) }
-    val defFile = arguments.def?.let { File(it) }
-    val manifestAddend = arguments.manifest?.let { File(it) }
+    val defFile = argParser.get<String>("def")?.let { File(it) }
+    val manifestAddend = argParser.get<String>("manifest")?.let { File(it) }
 
-    if (defFile == null && arguments.pkg == null) {
-        usage()
-        return null
+    if (defFile == null && argParser.get<String>("pkg") == null) {
+        argParser.printError("-def or -pkg should provided!")
```

Comment:
Does it show help in this case as before?

Reply:
yes